xcodeproj/internal/bazel_integration_files/calculate_output_groups.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/python3
+
+import glob
+import json
+import os
+import sys
+import time
+import traceback
+
+def _calculate_build_request_file(objroot):
```

Comment:
I'll admit ahead of time that there should be some unit tests for this file. Maybe before 1.0 we can get to that, but for now I'm relying on the fact that it does the fallback logic in the case of errors, and people will really notice if it's not working.